.github/workflows/reset-environment.yml

```diff
@@ -0,0 +1,219 @@
+        required: true
+    inputs:
+      # This needs to be stringified JSON because inputs don't support arrays, see 
+      # https://github.community/t/reusable-workflow-with-strategy-matrix/205676/2.
+      machine-types:
+        required: false
+        type: string
+        default: "[\"ubuntu-latest\"]"
+        description: >
+            Stringified JSON array with the name of the type of machine(s) to run the workflow under, e.g.
+            "[\"ubuntu-latest\"]" or "[\"ubuntu-latest\", \"windows-latest\"]".
+      timeout-minutes:
+        required: false
+        type: number
+        default: 360
+        description: Configuration for the timeout-minutes parameter of the workflow. The 360 is GitHub's default.
+      app-name:
+        required: true
+        type: string
+        description: What you see at the top of the blade on the Azure Portal. Can contain uppercase letters too.
+      destination-slot-name:
+        required: false
+        type: string
+        default: Staging
+        description: The slot name of the web app you want to reset. What you see at the top of the blade on the Azure
+          Portal, when you open the slot, before the app name in parenthesis.
+      source-slot-name:
+        required: false
+        type: string
+        default: Production
+        description: The slot name of the web app you want to reset the destination web app with. What you see at the
+          top of the blade on the Azure Portal, when you open the slot, before the app name in parenthesis.
+      resource-group-name:
+        required: true
+        type: string
+        description: Name of the resource group.
+      url:
+        required: true
+        type: string
+        description: The URL of the web app slot.
+      database-connection-string-name:
+        required: true
+        type: string
+        description: The name of the database connection string.
+      database-master-connection-string-name:
+        required: true
+        type: string
+        description: The name of the database master connection string.
+      storage-connection-string-name:
+        required: true
+        type: string
+        description: The name of the storage connection string.
+      service-objective-name:
+        required: false
+        type: string
+        default: S0
+        description: The name of the service objective to scale the database to, e.g. B, S1, S4, P4, P6.
+      # These need to be stringified JSON because inputs don't support arrays, see 
+      # https://github.community/t/reusable-workflow-with-strategy-matrix/205676/2.
+      container-exclude-list:
+        required: false
+        type: string
+        default: "@()"
+        description: >
+            PowerShell string array with the name of the excluded container(s), e.g.
+            "@(\"not-media\")" or "@(\"not-media\", \"bad-container\")". The parameter must be a PowerShell string
+            array.
+      container-white-list:
```

Comment:
The opposite of an exclude list is an include list. Same for all the other cases here too.